README.md

```diff
@@ -4,11 +4,7 @@ Beat Saber open replay format.
 
 # BSOR V1
 
-- [C# code](https://github.com/BeatLeader/beatleader-mod/blob/master/Source/2_Core/Models/Replay.cs)
-- [JS code](https://github.com/BeatLeader/BeatSaber-Web-Replays/blob/master/src/open-replay-decoder.js)
```

Comment:
Why remove all the non-C# implementations?

Reply:
the links to them don't work anymore and I couldn't find the code in the repo